redi-connect-front/src/types/Language.ts

```diff
@@ -1 +1,4 @@
-export type Language = string;
+export type Language = {
```

Comment:
This looks like a good change but I don't think we should go about changing the data layer just yet. This change in particular would mean we need to update the admin, the backend and run a migration on all data in the database. Also, we don't know what other changes will be needed when/if we plug all data into salesforce. Perhaps they use other language/locale codes, etc.

Since this change seems to be mostly about changing the data structure to satisfy the form's expected data model, rather create some kind of simple adapter like this:
```
const languages = ['French', 'German', 'Farsi']
const formLangauges = languages.map(language => ({ id: language, label: language }))
```

Reply:
Ive noticed that the select can come in two forms. One is when there is only an array of strings and an array of objects. 
The idea is to unify this, for me its fine both ways, the question is, where the logic should sit. I can also put it to the end (select component)... however, I don't really like a lot of conditions in the frontend makes things hard to understand.

Reply:
True, there are two models, unfortunate early poor design decision that has become ingrained by now. Until we resolve it during the salesforce project, perhaps the most clean place to insert the adapter logic is right after the import of the languages array? Something like:

```
import { languages } from 'config.ts`
const formLangauges = languages.map(language => ({ id: language, label: language }))
```